CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ endif ()
 message(STATUS "CMAKE_INSTALL_PREFIX: ${CMAKE_INSTALL_PREFIX}")
 
 cmake_minimum_required(VERSION 3.14)
-project(MMDeploy VERSION 0.10.0)
+project(MMDeploy VERSION 0.12.0)
```

Comment:
Should it be 1.0.0?

Reply:
Since we are in 1.0.0rc{x} stage, I think it's not appropriate to label the version as `1.0.0`.
I think It's also not a good idea to set it as `1.0.0rc{x}`, otherwise the name of SDK library will be `libmmdeploy.so.1.0.0rc{x}`
Thus I choose to keep SDK version as the same as the master's.